src/components/FormPageComponents/NewAppBar.js

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import { Grid } from "@mui/material";
+import Pages from "./PageList";
+import Logo from "./Logo";
+import NavMenuComponent from "./NavMenu";
+
+const ResponsiveAppBar = () => {
+  return (
+    <AppBar position="sticky" color="secondary">
+      <Grid container direction="row" alignItems="center" test-id="xd">
+        <Toolbar>
+          <Grid item xs={2} sx={{
```

Comment:
this make no sense.

`<Grid
            item
            xs={2}
            sx={{
              display: {
                xs: "flex",
                md: "none",
              },
            }}
          >
            <Logo />
          </Grid>
          <Grid item xs={2} sx={{ display: { xs: "none", md: "flex" } }}>
            <Logo />
          </Grid>` = `          <Grid
            item
            xs={2}
            sx={{
              display: "fles",
            }}
          >
            <Logo />
          </Grid>`